Cargo.lock

```diff
@@ -493,14 +493,15 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "chrono"
-version = "0.4.19"
+version = "0.4.20"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "670ad68c9088c2a963aaa298cb369688cf3f9465ce5e2d4ca10e6e0098a1ce73"
+checksum = "6127248204b9aba09a362f6c930ef6a78f2c1b2215f8a7b398c06e1083f17af0"
 dependencies = [
- "libc",
+ "js-sys",
```

Comment:
Why is `js-sys` a dependency on what should just be a wrapper around `localtime_r`?

Reply:
Looks like `chronos` by default includes all the dependencies needed to compile to a wasm target.  Let's turn this off:

```
chrono = { version = "0.4.22", features = ["clock"] }
```

Reply:
Tried this and doesn't quite work as expected -- 0.4.22 still pulls in js-sys, iana-time-zone as well as android_system_properties.